{{cookiecutter.github_project_name}}/src/plugin.ts

```diff
@@ -37,6 +35,9 @@ function activateWidgetExtension(
   registry.registerWidget({
     name: MODULE_NAME,
     version: MODULE_VERSION,
-    exports: widgetExports,
+    exports: async () => import(
+      /* webpackChunkName: "{{ cookiecutter.npm_package_name }}" */ 
```

Comment:
This should be set to a different name to work - just tested locally on a private repo

```suggestion
      /* webpackChunkName: "{{ cookiecutter.npm_package_name }}-widgets" */ 
```

Reply:
Yes, I noticed that with threejs as well https://github.com/jupyter-widgets/pythreejs/pull/342#discussion_r538606446

Reply:
thx!

Reply:
Small note: playing a bit with this. It works by also dropping the all comment. As seen in the example below:

Before:
**asset mywidgets-model.43ed8686bb5736ca8d4e.js?v=43ed8686bb5736ca8d4e 20.4 KiB [emitted] [immutable] [minimized] (name: mywidgets-widgets)**
asset remoteEntry.2c68346fa273f734692b.js 11.3 KiB [emitted] [immutable] [minimized] (name: mywidgets)

After:
**asset 354.9d72a8586f604ab5e58e.js?v=9d72a8586f604ab5e58e 20.4 KiB [emitted] [immutable] [minimized]**
asset remoteEntry.766482559caf7cba28cf.js 11.3 KiB [emitted] [immutable] [minimized] (name: mywidgets)

In my case I got 14 chunks. So I'm not sure naming one of them beside the entry point is really useful.

What do you think?